integration/client/testdata/default-1.6.toml

```diff
@@ -79,6 +79,7 @@ version = 2
       conf_template = ""
       ip_pref = ""
       max_conf_num = 1
+      use_internal_loopback = false
```

Comment:
Oh, actually i don't think this file and the 1.7 toml should be updated, unless there is plan to backport this feature to those versions?

Reply:
I believe we should backport it however we don't need to. @mikebrow thoughts? @aojea

Reply:
nod saw that at the same time in my review :-) see comments..

I wouldn't backport it because I agree "we don't need to" and that has been the measure by which we make such decisions :-)